resources/views/mship/waiting-lists/view.blade.php

```diff
@@ -89,7 +89,7 @@
 </div>
 @if($list->isATCList() || count($automaticFlags))
 <div class="alert alert-warning">
-    <strong>Important: </strong> Automated eligibility flags are only calculated every 24 hours! If you have just completed
+    <strong>Important: </strong> Automated eligibility flags are only calculated every 24 hours, or after the end of an ATC session! If you have just completed
     a network session, the flags shown above may not be accurate.
```

Comment:
Might need to remove the last part of this paragraph if we imply it does update after ATC session? Or is it being kept to say it might not be instantaneous

Reply:
amended slightly